### ISSUE TITLE
Chaos created from front-end doesn't have labels

### ISSUE THREAD

**Issue by yeya24** [pingcap/chaos-mesh#637]
## Bug Report

**What did you do?**

Created a chaos with label specified, but it doesn't have any labels after creation.

![Screenshot from 2020-06-17 12-15-10](https://user-images.githubusercontent.com/25150124/84922759-50795a00-b094-11ea-9258-9c99fa2a84eb.png)

No labels here in the created chaos. Seems a problem at frontend. @g1eny0ung 

![Screenshot from 2020-06-17 12-16-29](https://user-images.githubusercontent.com/25150124/84922831-6b4bce80-b094-11ea-85d9-9f293fc4acb0.png)

**What did you expect to see?**

**What did you see instead?**

**Comment by g1eny0ung** [comment]
Because {app: etcd} is not a valid json and {"app": "etcd"} is.

After #634 merged into master, we can change this ''ugly" JSON input to a label selector. 😢

**Comment by yeya24** [comment]
I am wondering if there is a case that we want to create a chaos beforehand, I mean before the creation of the pods we want to test?

**Comment by yeya24** [comment]
And again, I test with {"app": "etcd"}, but it still doesn't have the label.

**Comment by g1eny0ung** [comment]
Oh, know about it, it's the `label selectors` not `labels`...

**Comment by yeya24** [comment]
> Oh, know about it, it's the `label selectors` not `labels`...

Yeah, sorry... Missed that, thanks!

**Comment by g1eny0ung** [comment]
Ha, never mind.

**Comment by g1eny0ung** [comment]
> I am wondering if there is a case that we want to create a chaos beforehand, I mean before the creation of the pods we want to test?

I am not sure, @cwen0 PTAL.

**Comment by cwen0** [comment]
> > I am wondering if there is a case that we want to create a chaos beforehand, I mean before the creation of the pods we want to test?
> 
> @yeya24 I am not sure, @cwen0 PTAL.

This is possible, but we do not recommend this method

**Comment by yeya24** [comment]
> > > I am wondering if there is a case that we want to create a chaos beforehand, I mean before the creation of the pods we want to test?
> > 
> > 
> > @yeya24 I am not sure, @cwen0 PTAL.
> 
> This is possible, but we do not recommend this way to use chaos-mesh

Yeah, sounds reasonable. I will close this issue now.
